apps/backoffice-v2/src/common/components/organisms/Filters/filter-inputs/MultiSelectFilterInput.tsx

```diff
@@ -0,0 +1,27 @@
+import { MultiSelect } from '@/common/components/atoms/MultiSelect/MultiSelect';
+import { IFilterInputParams } from '@/common/components/organisms/Filters/interfaces';
+import { FunctionComponent } from 'react';
+
+export interface IMultiSelectFilterParams {
+  type: 'multi-select';
+  options: Array<{
+    name: string;
+    value: string;
+  }>;
+}
+
+export const MultiselectFilterInput: FunctionComponent<
+  IFilterInputParams<IMultiSelectFilterParams, string[]>
+> = ({ inputParams, filter, value, onChange, clear }) => {
+  const { options } = inputParams;
+
+  return (
+    <MultiSelect
+      title={filter.label}
+      selectedValues={value ?? []}
+      onSelect={onChange}
+      onClearSelect={clear}
+      options={options || []}
+    />
+  );
+};
```

Comment:
Why do we need this abstraction?

Reply:
Its an adapter interface between input components and generic filter interfaces so all types can be consistent except specific params such as `options`.